Loop/Managers/LoopDataManager.swift

```diff
@@ -198,12 +198,13 @@ final class LoopDataManager {
 
         if insulinEffect == nil {
             updateGroup.enter()
-            deviceDataManager.doseStore.getGlucoseEffects(startDate: effectStartDate) { (effects, error) -> Void in
-                if let error = error {
+            deviceDataManager.doseStore.getGlucoseEffects(start: effectStartDate, end: .distantFuture) { result -> Void in
```

Comment:
The changes here, in StatusExtensionDataManager, PredictionTableViewController, and StatusTableViewController don't appear to be related to swift 3.1.  Could you put those into a different PR along with a description of what they're addressing?